test/unit/mocks.js

```diff
@@ -1,18 +1,21 @@
-define(["ItemsHoldr"], function (ItemsHoldrModule) {
+define(["ItemsHoldr", "ItemValue"], function (ItemsHoldrModule, ItemValueModule) {
     var ItemsHoldr = ItemsHoldrModule.ItemsHoldr;
+    var ItemValue = ItemValueModule.ItemValue;
 
     var mocks = {
         /**
-         * 
+         * @param [settings]   Settings for the ItemValue.
+         * @returns An ItemValue instance.
          */
-        mockItemValue: function (settings) {
-            return new ItemValue(settings)
+        mockItemValue: function (IH, key, settings) {
```

Comment:
Can you update the JSDoc for this? I'm also wary of names like `IH`; better to spell it out.